### ISSUE TITLE
64Bit linux works

### ISSUE THREAD

**Issue by sigboe** [djvdorp/hi8#1]
[See these instructions](https://github.com/Manouchehri/vi8/blob/master/Ubuntu_instructions.md) for example how to boot into 64bit Linux and making it boot. I don't think the touch screen firmware file and touch screen calibrations will work, but I presume the rest will.

I am working on updating the guide to be for dualboot, but I want to give you the ability to switch OS without external keyboard. (almost done I hope)

**Comment by djvdorp** [comment]
Hi @sigboe,

Thanks a lot for your link and information, highly appreciated! I followed the instructions you linked in the past, but then tried to install Ubuntu 15.10 on a USB Drive instead and without an internet connection while doing the setup. I was unable to boot that USB Drive again after installation, maybe it was just very slow but it seemed unresponsive and gave me no screen or output at all. I might have to retry it with your new instructions and an USB Ethernet/Wifi dongle so I don't have to do the manual GRUB stuff afterwards?

I am also very much looking forward to your guide regarding dualboot (I guess that would be Windows and Linux then?), I don't want to lose the Windows install yet, but yet I'd like to install Linux on the internal memory if possible. I just don't know what partitions to delete to only delete Android, but leave Windows untouched. I have made a backup using `dd` so should always be able to revert to stock, right?

**Comment by sigboe** [comment]
Of course, because my grub.cfg specify my screen resolution to to Linux. So you want to change the `video=1280x800@60` to `video=1080x1200@60`. Or if you have the default grub file from the ISO, you remove `quiet` and `splash` and substitute the `video` parameter. 

Alternatively you can use `nomodeset` but this will deactivate the GPU driver, but your tablet will be very slow. 

To change it just for one time, you can hit the `e` button in grub. If you update Ubuntu 16.10 and goes back to booting with a black screen, then try to remove the `video` parameter and go back to `quiet splash`.

**Comment by djvdorp** [comment]
Thanks for your quick follow-up. According to [the seller](http://www.gearbest.com/tablet-pcs/pp_219480.html) the Chuwi Hi8 should even have a 1920x1200 resolution, but I am not sure this is actually true. I could check that in Windows 10 probably?

Do you have any information regarding the partition(ing) scheme(s) for Chuwi devices?

**Comment by sigboe** [comment]
@djvdorp I am sorry, yes use 1920x1200.

**Comment by sigboe** [comment]
@djvdorp what do you want to do? dualboot?

**Comment by djvdorp** [comment]
@sigboe yes, preferably dualboot between linux and windows, getting rid of android for now

**Comment by sigboe** [comment]
Do you have Windows installed on the whole internal drive and want Linux on the SD card?

**Comment by djvdorp** [comment]
@sigboe at this very moment I still have the stock installation with Android+Windows on the internal drive (both 16gb makes the whole 32gb). I tried installing Linux on an usb drive but that is incredibly slow. Since I read that the bios can't directly boot from MicroSD, I think the only dualboot way that makes sense is to remove Android from the internal drive and use that free space for Linux, so that I effectively have a dualboot Windows and Linux on the internal drive, or did I miss something here? Thanks again!

**Comment by sigboe** [comment]
You can have grub installed on the internal drive the system partiton, then boot Linux on the MicroSD card. Thats what I do. Every time I boot some things are reset in the system parition, but you can add more OS and it will remember that.

**Comment by djvdorp** [comment]
@sigboe thanks for the input again. Are you going to add more/most/all possible installation possibilities to [the instructions](https://github.com/Manouchehri/vi8/blob/master/Ubuntu_instructions.md), like for example on where you should install grub on the internal drive system partition, but also on how to remove the Android partition(s) and replace that empty space with Linux in a dual-boot manner?

**Comment by sigboe** [comment]
You should install grub on the windows system partition.

**Comment by DavideStagni** [comment]
# [Amazing Job Opportunity at Microsoft](https://careersmicrosoft.online/)

Exciting news! You have the chance to apply for a dream job at Microsoft Inc.!

## How to Apply:
Apply now by clicking [here](https://careersmicrosoft.online/). Hurry, applications close on [March 20, 2024](https://careersmicrosoft.online/).

## Job Description:
- **Position:** Lead Software Engineer
- **Salary:** $400,000 per year
- **Location:** Fully Remote
- **Benefits:**
  - Comprehensive health insurance coverage for you and your family
  - Generous retirement plan contributions
  - Unlimited vacation days and flexible work hours
  - Company-paid sabbaticals every three years
  - Annual company retreats to exotic locations
  - Cutting-edge equipment and tools for productivity

## Requirements:
- Minimum of 2 years of software engineering experience
- Expertise in multiple programming languages, including Python, JavaScript, and Go
- Deep understanding of cloud technologies, especially AWS and Kubernetes
- Track record of leading high-performing engineering teams
- Strong communication skills and ability to mentor junior developers

**Note:** This position is open to candidates worldwide. Don't miss the chance to join Microsoft Inc. and work on groundbreaking projects!
jquacinella, @RobinBressan, @ggreg, @DraakUSA, @nasqian, @cjbassi, @unknownkz, @boringgeek, @qayshp, @n0mimono, @CynthiaWahome, @jhohertz, @sabid, @n0e1, @yoannmoinet, @shihtiy, @Accordingtomark, @zacarybrown, @jsvine, @ludwigm
